src/Mvc/Mvc.ApiExplorer/test/EndpointMetadataApiDescriptionProviderTest.cs

```diff
@@ -644,6 +644,44 @@ public void HandleAcceptsMetadataWithTypeParameter()
             Assert.True(bodyParameterDescription.IsRequired);
         }
 
+        [Fact]
+        public void FavorsProducesMetadataOverAttribute()
+        {
+            // System.Console.WriteLine($"Waiting for debugger to attach on {System.Environment.ProcessId}");
+            // while (!System.Diagnostics.Debugger.IsAttached)
+            // {
+            //     Thread.Sleep(1000);
+            // }
+            // System.Console.WriteLine("Debugger attached");
```

Comment:
Remove

Reply:
Ooof -- I need a git hook to check for these. #justvscodeproblems